run_lualatex.py

```diff
@@ -66,7 +66,7 @@
 return_code = subprocess.call(
     args=[
         latexrun_file,
-        "--latex-args=-shell-escape -jobname=" + job_name,
+        "--latex-args=-shell-escape -jobname='" + job_name.replace("'", "'\\''") + "'",
```

Comment:
Should we use something a bit more robust here, like [shlex.quote()](https://docs.python.org/3/library/shlex.html#shlex.quote) or [pipes.quote()](https://docs.python.org/2/library/pipes.html#pipes.quote) if we have to support Python 2 (I'm really not familiar with Python and don't know how to tell from a quick glance whether this is Python 2 or 3...)

Reply:
Unless I'm missing something, we're using Python 3 (https://bazel.build/reference/be/python#py_binary.python_version). `shlex.quote()` would likely be the best way forward here.